### DISCUSSION TITLE
Unable to connect to a user's phone

### DISCUSSION THREAD

**Discussion by jbrines** [rustdesk/rustdesk#3340] [General]
Hi Guys,

I  got a user to install the app on their Samsung Phone, got them to share their screen and start service. When I added their ID in to my desktop app it would not connect and no request would pop up on the phone.

Also another user when I tried to connect the desktop app came back with "reset by peer" but there is nothing showing on the phone.

I have tried it on my own phone and connect without any issue.

**Comment by rustdesk** [comment]
try out nightly build

**Reply by rustdesk** [reply]
If still fail, try add a `/r` suffix to the id. e.g. your id is `12344`, please intpu `12344/r` in the id input box.

**Reply by rustdesk** [reply]
or toggle this.

![image](https://user-images.githubusercontent.com/71636191/220914973-bd32b1e7-62c8-4430-97b5-7477d665375d.png)

**Reply by jbrines** [reply]
> or toggle this.
> 
> ![image](https://user-images.githubusercontent.com/71636191/220914973-bd32b1e7-62c8-4430-97b5-7477d665375d.png)

Where is this on the desktop client? I am using Windows Version 1.1.9

**Reply by jbrines** [reply]
> try out nightly build

That appears to be working
